EIPS/eip-2364.md

```diff
@@ -40,15 +41,17 @@ Whenever two peers connect using the `eth/64` protocol, the updated `Status` mes
 
 ## Rationale
 
-##### EIP-2124 mentions advertising the `forkid` in the discovery protocol too. How does that compare to advertising in the `eth` protocol? Why is the redundancy needed?
+The specification is tiny since most parts are already specified in EIP-2124. `eth/63` is not specified as an EIP, but is maintained [here](https://github.com/ethereum/devp2p/blob/master/caps/eth.md).
```

Comment:
Not sure why this needs to be here? Is this the way to make @MicahZoltu allow an external URL? :)

Reply:
And can this URL be a tagged one, and not `master`?

Reply:
Ah, you are right, that should be removed.  It is not necessary for the specification.